File: .config/hiss/testdata/HISS-04/c/gap/cognitive-high.c
/* Cognitive complexity 28 (seven nesting levels) in 18 lines. */
int deeply_nested(int count) {
    if (count > 0) {
        if (count > 1) {
            if (count > 2) {
                if (count > 3) {
                    if (count > 4) {
                        if (count > 5) {
                            if (count > 6) {
                                count++;
                            }
                        }
                    }
                }
            }
        }
    }
    return count;
}

File: .config/hiss/testdata/HISS-04/python/gap/cognitive-high.py
# Cognitive complexity 28 (seven nesting levels) in 11 lines.
def deeply_nested(count):
    if count > 0:
        if count > 1:
            if count > 2:
                if count > 3:
                    if count > 4:
                        if count > 5:
                            if count > 6:
                                count += 1
    return count
